packages/vc-data/src/account/v1.ts

```diff
@@ -97,7 +97,7 @@ export type AccountV1 = CreateThing<
     accountTypeConfidence?: number
     accountStatements?: Array<AccountStatementV1>
     accountPayments?: Array<AccountPaymentV1>
-    hasValue?: ExpandThing<MonetaryAmountRV1>
+    hasValue?: OneOrMore<MonetaryAmountRV1 | MonetaryAmountEV1>
```

Comment:
You'll probably want `OneOrMore<ExpandThing<MonetaryAmountRV1> | ExpandThing<MonetaryAmountEV1>>`